pkg/operation/hybridbotanist/addons.go

```diff
@@ -148,11 +148,6 @@ func (b *HybridBotanist) generateCoreAddonsChart() (*chartrenderer.RenderedChart
 		return nil, err
 	}
 
-	ccmConfig := map[string]interface{}{}
```

Comment:
Can you also remove https://github.com/gardener/gardener/blob/master/charts/shoot-core/charts/cloud-controller-manager/values.yaml#L1?